From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 11 Sep 2025 13:27:35 +0200
Subject: build: Handle data as first subdir

We generate resources and files that may be required in next stages, so
lets do it as first thing

Applied-upstream: 49.0, commit:89b571e3cbcaa82e79274339dd015cd8c46d6fce
Applied-upstream: 48.5, commit:3710de43319528eefc5e31aef51c5d026ee45172
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 1f3c33f..58a69ef 100644
--- a/meson.build
+++ b/meson.build
@@ -316,10 +316,10 @@ endif
 
 po_dir = join_paths(meson.current_source_dir(), 'po')
 
+subdir('data')
 subdir('js')
 subdir('src')
 subdir('po')
-subdir('data')
 
 if get_option('tests') and have_x11_client
   subdir('tests')
